src/uglify/serialization.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable
+  no-new-func
+*/
+import espree from 'espree';
```

Comment:
The usage of espree and ESLint seems really really heavy handed here. I think we need to go back to the simpler approach of caching the error etc.